src/app/ApplicationTemplate.Shared/Client/Mock/AuthenticationEndpointMock.cs

```diff
@@ -18,9 +18,12 @@ public AuthenticationEndpointMock(IObjectSerializer serializer)
 			_serializer = serializer ?? throw new ArgumentNullException(nameof(serializer));
 		}
 
-		public async Task CreateAccount(CancellationToken ct, string email, string password)
+		public async Task<AuthenticationData> CreateAccount(CancellationToken ct, string email, string password)
 		{
 			await Task.Delay(TimeSpan.FromSeconds(2));
```

Comment:
Do we need the delay?

Reply:
If this delay simulates a long API call, simply add a comment that says so. :)